WebRoot/public/login.jsp

```diff
@@ -108,11 +108,14 @@
 		<c:when test="${param.loginOp eq 'logout'}">
 			<zm:getDomainInfo var="domainInfo" by="virtualHostname" value="${zm:getServerName(pageContext)}"/>
 			<c:set var="logoutRedirectUrl" value="${domainInfo.attrs.zimbraWebClientLogoutURL}" />
+			<c:set var="skipLogoff" value="${domainInfo.attrs.zimbraWebClientSkipLogoff}" />
```

Comment:
These changes are not from ZCS-11302, not sure from where they are coming, should I remove these lines as well?

Reply:
rebase issue, changes are from 8815p32

Reply:
done